lib/Migrate_ast.ml

```diff
@@ -150,7 +150,13 @@ module Location = struct
 
   let sexp_of_t x = Sexp.Atom (to_string x)
 
-  let compare : t -> t -> int = Poly.compare
+  let compare {loc_start; loc_end; loc_ghost} b =
+    match Position.compare loc_start b.loc_start with
+    | 0 -> (
+      match Position.compare loc_end b.loc_end with
+      | 0 -> Poly.compare loc_ghost b.loc_ghost
```

Comment:
```suggestion
      | 0 -> Bool.compare loc_ghost b.loc_ghost
```

We might as well get rid of this one too. FWIW, this function is showing up a lot in the profiles so I'm glad it's not using poly compare anymore